components/centos/centos-openshift-setup/Vagrantfile

```diff
@@ -115,8 +115,8 @@ Vagrant.configure(2) do |config|
     echo "Configured cluster admin user: admin, Password: admin"
     echo
     echo "To use OpenShift CLI, run:"
-    echo "$ vagrant ssh"
-    echo "$ oc login"
+    echo "vagrant service-manager install-cli openshift"
```

Comment:
Changed it to `To download OC binary : vagrant service-manager install-cli openshift`